projects/ngx-formentry/src/form-entry/value-adapters/diagnosis.adapter.ts

```diff
@@ -0,0 +1,175 @@
+import {Injectable} from '@angular/core';
+import {Form} from '../form-factory/form';
+import {ValueAdapter} from './value.adapter';
+
+@Injectable()
+export class DiagnosisValueAdapter implements ValueAdapter {
+  formDiagnosisNodes = [];
```

Comment:
I'm not sure I see the utility of having a class level property for this?

Reply:
I can remove it but will make my life a lot harder while testing.